### ISSUE TITLE
NVDA doesn't read the fields in Windows LiveWriter

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1709]
**Reported by danielgartmann on 2011-08-06 23:08**
When composing a new blog entry, there is a field for title and another field for writing the text of the post. NVDA doesn't let me edit either of these fields, not even when I force NVDA to use Forms Mode.

**Comment by leonardder** [comment]
I'm not sure whether Windows Live writer is still alive, but given the state of Windows Live in general and the abandoned state of this ticket, I'm closing this for now as a won't fix. Please feel free to request a reopen and, if so, please provide the steps to reproduce this.
